nibe/connection/__init__.py

```diff
@@ -93,4 +98,6 @@ async def verify_connectivity_read_write_alarm(
     if coil.mappings:
         value = coil.mappings[str(value)]
     coil_data.value = value
-    await connection.write_coil(coil_data)
+
+    with suppress(WriteDeniedException):
+        await connection.write_coil(coil_data)
```

Comment:
Maybe we need to at least log these?

Reply:
Not really i think. This check is just about verifying connectivity. If i get a denied response, that is just as well as a successfull one.

Reply:
It is still possible to set a wrong port number. That would most likely lead to the same error right?
On the other hand. If there is no read error, at least the device works, IP is set correctly etc.

Reply:
Nah, that will not lead to denied, that will lead to a timeout.